src/EnergyPlus/HybridEvapCoolingModel.cc

```diff
@@ -0,0 +1,1591 @@
+				MinSAT = GetCurrentScheduleValue(TsaMin_schedule_pointer);
+			}
+			if (TsaMax_schedule_pointer > 0) {
+				MaxSAT = GetCurrentScheduleValue(TsaMax_schedule_pointer);
+			}
+			if (Tosa < MinSAT || Tosa > MaxSAT)
+				return false;
+			return true;
+		}
+
+		bool Model::MeetsSupplyAirRHOC(double Wosa)
+		{
+			// implement
+			if (Wosa < 0)
+				//temp fix
+				Wosa = 0.003;
+			//return false;
+			return true;
+		}
+
+		Model::Model() :
+			//Tsa_curve_pointer(0), HRsa_curve_pointer(0),Psa_curve_pointer(0),
+			TsaMin_schedule_pointer(0),
+			TsaMax_schedule_pointer(0),
+			RHsaMin_schedule_pointer(0),
+			RHsaMax_schedule_pointer(0),
+			ZoneNum(0),
+			SystemMaximumSupplyAirFlowRate(0),
+			ScaledSystemMaximumSupplyAirMassFlowRate(0),
+			ScalingFactor(0),
+			SchedPtr(0),
+			UnitTotalCoolingRate(0.0),
+			UnitTotalCoolingEnergy(0.0),
+			UnitSensibleCoolingRate(0.0),
+			UnitSensibleCoolingEnergy(0.0),
+			UnitLatentCoolingRate(0.0),
+			UnitLatentCoolingEnergy(0.0),
+			SystemTotalCoolingRate(0.0),
+			SystemTotalCoolingEnergy(0.0),
+			SystemSensibleCoolingRate(0.0),
+			SystemSensibleCoolingEnergy(0.0),
+			SystemLatentCoolingRate(0.0),
+			SystemLatentCoolingEnergy(0.0),
+			UnitTotalHeatingRate(0.0),
+			UnitTotalHeatingEnergy(0.0),
+			UnitSensibleHeatingRate(0.0),
+			UnitSensibleHeatingEnergy(0.0),
+			UnitLatentHeatingRate(0.0),
+			UnitLatentHeatingEnergy(0.0),
+			SystemTotalHeatingRate(0.0),
+			SystemTotalHeatingEnergy(0.0),
+			SystemSensibleHeatingRate(0.0),
+			SystemSensibleHeatingEnergy(0.0),
+			SystemLatentHeatingRate(0.0),
+			SystemLatentHeatingEnergy(0.0),
+			RequestedLoadToHeatingSetpoint(0.0),
+			RequestedLoadToCoolingSetpoint(0.0),
+			RequestedHumdificationMass(0.0),
+		RequestedHumdificationLoad(0.0),
+		RequestedHumdificationEnergy(0.0),	
+			RequestedDeHumdificationMass(0.0),
+			RequestedDeHumdificationLoad(0.0),
+			RequestedDeHumdificationEnergy(0.0),
+			PrimaryMode(0),
+			PrimaryModeRuntimeFraction(0.0),
+			ErrorCode(0),
+			InletNode(0),
+			OutletNode(0),
+			SecondaryInletNode(0),
+			SecondaryOutletNode(0),
+			InletMassFlowRate(0.0),
+			InletTemp(0.0),
+			InletWetBulbTemp(0.0),
+			InletHumRat(0.0),
+			InletEnthalpy(0.0),
+			InletPressure(0.0),
+			InletRH(0.0),
+			OutletMassFlowRate(0.0),
+			OutletVolumetricFlowRate(0.0),
+			OutletTemp(0.0),
+			OutletWetBulbTemp(0.0),
+			OutletHumRat(0.0),
+			OutletEnthalpy(0.0),
+			OutletPressure(0.0),
+			OutletRH(0.0),
+			SecInletMassFlowRate(0.0),
+			SecInletTemp(0.0),
+			SecInletWetBulbTemp(0.0),
+			SecInletHumRat(0.0),
+			SecInletEnthalpy(0.0),
+			SecInletPressure(0.0),
+			SecInletRH(0.0),
+			SecOutletMassFlowRate(0.0),
+			SecOutletTemp(0.0),
+			SecOutletWetBulbTemp(0.0),
+			SecOutletHumRat(0.0),
+			SecOutletEnthalpy(0.0),
+			SecOutletPressure(0.0),
+			SecOutletRH(0.0),
+			Wsa(0.0), FinalElectricalPower(0.0), FinalElectricalEnergy(0.0), SupplyVentilationAir(0.0), SupplyVentilationVolume(0.0),
```

Comment:
A couple indentation issues above, more whitespace (padding) issues, and then split this line up so the style matches everywhere else.